Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -50,13 +50,31 @@ public extension Customizable where Self: UIViewController {
     }
 }
 
-extension UIConfigProvider where Self: _View {
-    public func uiConfigDidRegister() {
+extension ComponentsProvider where Self: _View {
+    public func componentsDidRegister() {
         if isInitialized {
             log.assertionFailure(
-                "`UIConfig` was assigned after the view has been already initialized. This is most likely caused by assigning " +
-                    "the custom `UIConfig` instance after the view has been added to the view hierarchy, or after the view's subviews " +
-                    "have been initialized already. This is undefined behavior and should be avoided."
+                """
+                `Components` was assigned after the view has been already initialized. \
+                This is most likely caused by assigning the custom `Components` instance \
+                after the view has been added to the view hierarchy, or after the view's subviews \
+                have been initialized already. This is undefined behavior and should be avoided.
```

Comment:
why the `\` characters? 🤔

Reply:
This is actually pretty clever and we should be using it anywhere where the text is too long :) 

Taken from Swift docs: 

>When your source code includes a line break inside of a multiline string literal, that line break also appears in the string’s value. If you want to use line breaks to make your source code easier to read, but you don’t want the line breaks to be part of the string’s value, write a backslash (\) at the end of those lines:

https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html

Reply:
🤯 I had no idea about this 🤯 nice 👍

Reply:
These new lines were the reason why we used the string concatenating for cases like this. Awesome 👍

Reply:
But why in this case don't we want the line breaks? 🤔 IMO it would be easier to read?